content/docs/command-reference/plots/diff.md

```diff
@@ -63,7 +63,8 @@ all the current plots, without comparisons.
   ```
 
 - `-o <path>, --out <path>` - specify a directory to write the HTML file
-  containing the plots (`dvc_plots/` by default).
+  containing the plots. The default is `dvc_plots` or the value set with the
+  [`plots.out_dir`](/doc/command-reference/config#plots) config option.
```

Comment:
Does it create the path if it doesn't exist?

Reply:
Yes